dynamic_programming/fizz_buzz.py

```diff
@@ -0,0 +1,72 @@
+# https://en.wikipedia.org/wiki/Fizz_buzz#Programming
+
+
+def fizz_buzz(
+    number: int, out: str, iterations: int, fizz_number: int, buzz_number: int
```

Comment:
`out`, `fizz_numher` and `buzz_number` shouldn't be passed, as they are temp/const variables.

You could perhaps define ```py
FIZZ_NUMBER = 3
BUZZ_NUMBER = 5
```